src/main/java/com/google/flourbot/api/Bot.java

```diff
@@ -69,12 +72,28 @@ public void onEvent(@RequestBody JsonNode event) throws IOException, GeneralSecu
         break;
       case "MESSAGE":
         // Sends request to execution module
-        String email = event.at("/message/sender/email").asText();
         String message = event.at("/message/text").asText();
         String threadId = event.at("/message/thread/name").asText();
-        replyText = macroExecutionModule.execute(email, message, threadId);
+
+        String[] words = message.split(" "); 
+
+        if (words[1].equalsIgnoreCase("initiate")) {
+            String creator = event.at("/message/sender/email").asText();
+            roomToCreator.put(event.at("/space/name").asText(), creator);
+            replyText = "Your macros have now been initiated for this room. All users in this room can use your macros.";
```

Comment:
There is some verification needed here. We don't know if the person who initiates the macro is in fact the creator, or if the macro even exists.